modules/widgets/internet-defense-league.php

```diff
@@ -0,0 +1,163 @@
+<?php /*
+
+**************************************************************************
+
+Plugin Name:  Internet Defense League
```

Comment:
I think we can remove this doc stuff.  The format is kinda weird and it's not included in other widgets.  Also not a plugin :)